source/gameengine/GameLogic/SCA_VibrationActuator.cpp

```diff
@@ -65,13 +68,35 @@ bool SCA_VibrationActuator::Update()
 	bool bPositiveEvent = IsPositiveEvent();
 
 	if (bPositiveEvent) {
-		instance->RumblePlay(m_strength, m_duration);
-		m_endtime = PIL_check_seconds_timer() * 1000.0f + m_duration;
+		switch (m_mode) {
+			case KX_ACT_VIBRATION_PLAY:
+			{
+				float strength[2];
```

Comment:
Is it an advantage to use an array as argument in `RumblePlay` if we have to pack the array in 3 lines before ?